helpers/wrapper/reports/slither.txt

```diff
@@ -0,0 +1,37 @@
+INFO:Detectors:
+Wrapper.swap(Types.Order) (Full.sol#456-506) ignores return value by low-level calls "msg.sender.call.value(_order.signer.param)()" (Full.sol#504)
+Reference: https://github.com/crytic/slither/wiki/Detector-Documentation#unchecked-low-level-calls
+INFO:Detectors:
+Wrapper.swap(Types.Order) (Full.sol#456-506) ignores return value by external calls "wethContract.transfer(_order.sender.wallet,_order.sender.param)" (Full.sol#480)
+Wrapper.swap(Types.Order) (Full.sol#456-506) ignores return value by external calls "wethContract.transferFrom(_order.sender.wallet,address(this),_order.signer.param)" (Full.sol#497)
+Reference: https://github.com/crytic/slither/wiki/Detector-Documentation#unused-return
+INFO:Detectors:
+Different versions of Solidity is used in :
+	- Version used: ['0.5.12', 'ABIEncoderV2', '^0.5.0']
+	- Full.sol#20 declares pragma solidity0.5.12
+	- Full.sol#21 declares pragma experimentalABIEncoderV2
+	- Full.sol#172 declares pragma solidity0.5.12
+	- Full.sol#295 declares pragma solidity0.5.12
+	- Full.sol#314 declares pragma solidity^0.5.0
+	- Full.sol#409 declares pragma solidity0.5.12
+Reference: https://github.com/crytic/slither/wiki/Detector-Documentation#different-pragma-directives-are-used
+INFO:Detectors:
+Pragma version "0.5.12" allows old versions (Full.sol#20)
+Pragma version "0.5.12" allows old versions (Full.sol#172)
```

Comment:
I know this is a slither output. Question about the actual output, not really the PR...
Does it? I thought the lack of `^` meant the version was fixed?

Reply:
It appears to complain if any version other than `0.5.3` is set. We should consider adding the caret to these to keep valid compilers under the `0.6.0`.